viewer-prototype/src/browser/trace-viewer/components/timegraph-output-component.tsx

```diff
@@ -318,8 +318,8 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
 
             if (value === -1) {
                 return {
-                    color: 0xCACACA,
-                    height: this.props.style.rowHeight * 0.5,
+                    color: style.color,
+                    height: style.height,
                     borderWidth: element.selected ? 2 : 0,
                     borderColor: 0xeef20c
```

Comment:
I don't think we need this 'if' block any more. Lines 345 - 350 should take care of this?

Reply:
Yes, thanks for pointing this out. Updated the code